src/audio/google/google_rtc_audio_processing.c

```diff
@@ -633,121 +633,139 @@ static int google_rtc_audio_processing_prepare(struct processing_module *mod,
 	return 0;
 }
 
+/* Extracts connected buffer pointers.  Clumsy, should cache in comp_data */
+static void find_bufs(struct processing_module *mod, struct comp_buffer **mic,
+		      struct comp_buffer **ref, struct comp_buffer **out)
+{
+	struct google_rtc_audio_processing_comp_data *cd = module_get_private_data(mod);
+	struct list_item *li;
+	int i = 0;
+
+	list_for_item(li, &mod->dev->bsource_list) {
+		struct comp_buffer *b = container_of(li, struct comp_buffer,
+						     sink_list);
+		if (i == cd->raw_microphone_source)
+			*mic = b;
+		else if (i == cd->aec_reference_source)
+			*ref = b;
+		i++;
+	}
+
```

Comment:
Should we do something when one of them (or both) is not found ?

Reply:
Probably?  That would reflect an unrecoverable topology bug.  But for sure it should be loggged.